trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.test.ts

```diff
@@ -0,0 +1,57 @@
+import { getBackLinkUrl } from './get-back-link-href';
+
+console.error = jest.fn();
+
+describe('getBackLinkUrl', () => {
+  describe('should return URL without params', () => {
+    it('when feeRecordIds array is empty', () => {
```

Comment:
I'm not personally a fan of this way around of splitting the when and should between the describe and it. Personally I wouldn't actually use a nested describe block here since there are only 4 tests.